services/py-api/tests/unit_tests/conftest.py

```diff
@@ -62,11 +67,16 @@ def participant_repo_mock() -> Mock:
     participant_repo.create = AsyncMock()
     participant_repo.delete = AsyncMock()
 
-    return participant_repo
+    yield participant_repo
+
+    participant_repo.reset_mock()
```

Comment:
Question for the way I understand it: So basically here were reset the mock so that we have a clear mock object from one test to the other right?

Reply:
Yea it's a good practice, I believe I have put a comment somewhere up the code.

Reply:
Fixed in 856b2d8a134a84cab904ff83ee54d0dea708d92f 
I removed it as it was redundant due to fixtures having `scope=function` by default. https://docs.pytest.org/en/stable/how-to/fixtures.html#fixture-scopes